### PR TITLE
Links to subject and age group pages

### PR DESCRIPTION
**Pull request by emmaachesongray**
### Trello card
https://trello.com/c/njLTjUOf/6465

### CODE REVIEW

File: app/views/content/steps-to-become-a-teacher/_step_4_find_a_teacher_training_course.html.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <p>Through teacher training, you can get qualified teacher status (QTS) or QTS with a postgraduate certificate in education (PGCE).</p>
 
-<p>You need QTS to teach in most primary and secondary schools in England but you do not need a PGCE to teach.</p>
+<p>You need QTS to teach in most primary and secondary schools in England but you do not need a PGCE to teach. Explore the <a href="/life-as-a-teacher/age-groups-and-specialisms">age groups and specialisms</a> you could qualify to teach</p>.
```
Comment (emmaachesongray) [inline app/views/content/steps-to-become-a-teacher/_step_4_find_a_teacher_training_course.html.erb:3]:
```suggestion
<p>You need QTS to teach in most primary and secondary schools in England but you do not need a PGCE to teach. Explore the <a href="/life-as-a-teacher/age-groups-and-specialisms">age groups and specialisms</a> you could qualify to teach.</p>
```

File: app/views/content/landing/how-to-become-a-teacher/_content.html.erb [hunk 2]
```diff
@@ -28,7 +28,7 @@
       </p>
 
       <p>
-        If you're interested in teaching certain subjects, you might be able to get a scholarship or bursary of up to <%= v :scholarships_generic_maxshortened %> tax-free.
+        If you're interested in <%= link_to("teaching certain subjects", page_path("life-as-a-teacher/explore-subjects")) %>, you might be able to get a scholarship or bursary of up to <%= v :scholarships_generic_maxshortened %> tax-free.
```
Comment (gemmadallmandfe) [inline app/views/content/landing/how-to-become-a-teacher/_content.html.erb:31]: I'm not sure this is a good link to include - we will be building out the explore subjects section to include non-bursary subjects?

This is also a paid search/social landing page so is noindexed - so not so relevant for SEO. Plus we tried to have a principle of only one signposting link per section on these pages!

Comment (emmaachesongray) [inline app/views/content/landing/how-to-become-a-teacher/_content.html.erb:31]:
```suggestion
        If you're interested in teaching certain subjects, you might be able to get a scholarship or bursary of up to <%= v :scholarships_generic_maxshortened %> tax-free.
```

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**